components/servo/Cargo.lock

```diff
@@ -2017,6 +2017,7 @@ dependencies = [
 name = "script_tests"
 version = "0.0.1"
 dependencies = [
+ "euclid 0.10.2 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
I'm not sure why that's happening, but I think my original comment was wrong and ports/cef/Cargo.lock does not need to be updated. Go ahead and ignore this.